cloud/features.rst

```diff
@@ -0,0 +1,18 @@
+Features
+########
+
+Replication model
+-----------------
+
+Citus Cloud runs with a different replication from outlined in Citus docs. Instead of using Citus replication we leverage Postgres streaming replication. Postgres streaming replication provides superior gurantees about the state of your data, and with years of experience managing it we take the hard work out of it for you. With Citus Cloud if you want the equivilent of replication_factor equal to 2, the most popular setting for Citus users then you simply enable HA at provisioning.
```

Comment:
* "with a different replication from [that] outlined"
* gurantees -> guarantees
* equivilent -> equivalent
* maybe parenthesize "the most popular setting for Citus users"